aexpect/remote.py

```diff
@@ -75,6 +75,20 @@ def __str__(self):
         return "%s    (output: %r)" % (self.msg, self.output)
 
 
+class TransferBadClientError(TransferError):
+
+    def __init__(self, client):
+        TransferError.__init__(self, None, None)
+        self.client = client
+
+    def __str__(self):
+        return "Unknown file copy client: '%s', valid values are scp and rss" % self.client
+
+
+class SCPError(TransferError):
```

Comment:
Do we need the per-protocol exception granularity?

Reply:
I think we already have a good granularity of protocols when it comes to different login/transfer operations so I see no reason not to have this granularity also for exception handling. If people don't bother about it, they can always catch the more general exception.

Reply:
sure, just asking :-)